apps/mobile/src/screens/quiz/quiz.tsx

```diff
@@ -0,0 +1,164 @@
+import {
+	BackgroundWrapper,
+	Button,
+	InfinitePager,
+	LoaderWrapper,
+	ScreenWrapper,
+	Text,
+	View,
+} from "~/libs/components/components";
+import { PREVIOUS_INDEX_OFFSET } from "~/libs/constants/constants";
+import { DataStatus, NumericalValue, RootScreenName } from "~/libs/enums/enums";
+import {
+	useAppDispatch,
+	useAppForm,
+	useAppSelector,
+	useCallback,
+	useEffect,
+	useNavigation,
+	useRef,
+} from "~/libs/hooks/hooks";
+import { globalStyles } from "~/libs/styles/styles";
+import {
+	type InfinitePagerImperativeApi,
+	type NativeStackNavigationProp,
+} from "~/libs/types/types";
+import { categoryAnswerSelectedValidationSchema } from "~/packages/quiz/quiz";
+import { actions as quizActions } from "~/slices/quiz/quiz";
+
+import { Content, Counter } from "./libs/components/components";
+import { QUIZ_FORM_DEFAULT_VALUES } from "./libs/constants/costants";
+import {
+	type QuizFormValues,
+	type RootStackParameterList,
+} from "./libs/types/types";
+import { styles } from "./styles";
+
+const Quiz: React.FC = () => {
+	const dispatch = useAppDispatch();
+	const infinitePagerReference = useRef<InfinitePagerImperativeApi | null>(
+		null,
+	);
+
+	const navigation =
+		useNavigation<NativeStackNavigationProp<RootStackParameterList>>();
+
+	const {
+		answersByQuestionIndex,
+		currentQuestion,
+		currentQuestionIndex,
+		dataStatus,
+		questions,
+	} = useAppSelector((state) => state.quiz);
+
+	const currentAnswer = answersByQuestionIndex[currentQuestionIndex] ?? "";
+	const totalQuestionsAmount = questions.length - PREVIOUS_INDEX_OFFSET;
```

Comment:
```suggestion
	const totalQuestionsAmount = questions.length;
```
Why do you subtract PREVIOUS_INDEX_OFFSET?

Reply:
I misunderstood and reverted the changes I made.

Reply:
Not fixed